Copyright (c) 2004, Jan Kneschke, incremental
All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are met:

- Redistributions of source code must retain the above copyright notice, this
list of conditions and the following disclaimer.

- Redistributions in binary form must reproduce the above copyright notice,
this list of conditions and the following disclaimer in the documentation
and/or other materials provided with the distribution.

- Neither the name of the 'incremental' nor the names of its contributors may
be used to endorse or promote products derived from this software without
specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
THE POSSIBILITY OF SUCH DAMAGE.

Date: Dec 28, 2017 4:28 PM

From: 5ffdf22fdb6c86b83d7aa38b61563e33afad0901 Jan 10, 2018 3:50 AM

Subject: [PATCH]RDKB-12395 : Upgrading  lighttpd to 1.4.45, adding openssl and zlib
	
	Incorporate security fixes in latest version.
	
Source: COMCAST

Upstream-Status: Pending

Signed-off-by: Prakash KAPPESANNANAGAPPA <Prakash_KAPPESANNANAGAPPA@comcast.com>
---

diff --git a/src/mod_authn_file.c b/src/mod_authn_file.c
index 023a58a..d1cb0db 100644
--- a/src/mod_authn_file.c
+++ b/src/mod_authn_file.c
@@ -16,6 +16,7 @@

 #include "base.h"

+#undef USE_OPENSSL
 #ifdef USE_OPENSSL
 #include "base64.h"
 #include <openssl/md4.h>
